tests/RobotFramework/tests/cumulocity/remote-access/test_remote_access_custom_root_cert_path.robot

```diff
@@ -0,0 +1,27 @@
+*** Settings ***
+Resource    ../../../resources/common.resource
+Library    Cumulocity
+Library    ThinEdgeIO
+
+Test Tags    theme:c8y    theme:troubleshooting    theme:plugins    adapter:docker
+Test Setup    Custom Setup
+Test Teardown    Get Logs
+
+*** Test Cases ***
+
+Execute ssh command with a custom root certificate path
+    ${KEY_FILE}=    Configure SSH
+    Add Remote Access Passthrough Configuration
+    ${stdout}=    Execute Remote Access Command    command=tedge --version    exp_exit_code=0    user=root    key_file=${KEY_FILE}
+    Should Match Regexp    ${stdout}    tedge .+
```

Comment:
It might even better just to use an echo statement, as then you can then simplify the the assertion

```suggestion
    ${stdout}=    Execute Remote Access Command    command=sh -c 'echo foobar'    exp_exit_code=0    user=root    key_file=${KEY_FILE}
    Should Contain    ${stdout}    foobar
```